x-pack/plugin/security/cli/src/main/java/org/elasticsearch/xpack/security/cli/AutoConfigureNode.java

```diff
@@ -552,6 +554,7 @@ protected void execute(Terminal terminal, OptionSet options, Environment env) th
         try {
             // final Environment to be used in the lambda below
             final Environment localFinalEnv = env;
+            final DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern("dd-MM-yyyy-HH:mm:ss z");
```

Comment:
Is there a way to avoid the Z marker? I suspect it's pretty unknown to many users. Something like UTC instead?
I'd also remove the hyphen between date and time.

Reply:
I can remove the timezone alltogether